composer.json

```diff
@@ -319,7 +319,8 @@
             },
             "drupal/inline_entity_form": {
                 "Fixes field label when a new entity is created inline (https://www.drupal.org/project/inline_entity_form/issues/2842744)":
-                "https://www.drupal.org/files/issues/inline_entity_form-no_label_required_field_with_no_entries-2842744.patch"
+                "https://www.drupal.org/files/issues/inline_entity_form-no_label_required_field_with_no_entries-2842744.patch",
+                "Add a setting to enable/disable inline editing of existing entities": "https://www.drupal.org/files/issues/2018-10-11/ief_edit_existing_2913571_7.patch"
```

Comment:
Please link to the comment where this file is uploaded. See other patches in this file for an example of how we usually do this. I know the info is in the filename but for consistency lets add it to the left hand side as well.